.lgtm.yml

```diff
@@ -0,0 +1,11 @@
+# docs: https://lgtm.com/help/lgtm/lgtm.yml-configuration-file
+
+path_classifiers:
+  test:
+    - exclude: python.d/python_modules/third_party/*
+    - exclude: python.d/python_modules/urllib3/*
+    - exclude: python.d/python_modules/pyyaml2/*
+    - exclude: python.d/python_modules/pyyaml3/*
+queries:
+  - exclude: cpp/command-line-injection
```

Comment:
don't remove it yet please.
we need this active to file the bug.

Reply:
applied work-around with https://github.com/firehol/netdata/pull/4218
as suggested by https://github.com/Semmle/ql/issues/199#issuecomment-421996400

So, @paulfantom you can remove the exclusion for `cpp/command-line-injection`